server.js

```diff
@@ -68,6 +68,22 @@ function getClasses(raceClasses) {
   return Promise.all(classesPromises);
 }
 
+app.get('/api/v1/faction/:name', async (request, response) => {
+  try {
+    const races = await database('races').where('faction', request.params.name).select();
+    if (races.length) {
+      response.status(200).json(races);
+    } else {
```

Comment:
2. If it's successful races array for entered faction is returned in response with status code `200`